load-tests/script.js

```diff
@@ -0,0 +1,136 @@
+
+  // K6 vars, overload in the command
+  iterations: 1,
+  vus: 1
+};
+
+// URL strings
+if (env === 'dev' || env === 'test') {
+  emailUrl = `https://ches-${env}.apps.silver.devops.gov.bc.ca/${emailUrl}`;
+  oidcUrl = `https://${env}.oidc.gov.bc.ca/${oidcUrl}`
+} else if (env === 'prod') {
+  emailUrl = `https://ches.nrs.gov.bc.ca/${emailUrl}`;
+  oidcUrl = `https://oidc.gov.bc.ca/${oidcUrl}`
+} else if (env && env.match(/pr-\d+/g)) {
+  emailUrl = `https://ches-dev-${env}.nrs.gov.bc.ca/${emailUrl}`;
+  oidcUrl = `https://dev.oidc.gov.bc.ca/${oidcUrl}`
+} else {
+  fail('No valid environment supplied. Supply "dev", "test", "prod", or "pr##".');
+}
+
+// -------------------------------------------------------------------------------------------------
+// 2. Setup
+// -------------------------------------------------------------------------------------------------
+export function setup() {
+  console.log(`Email url: ${emailUrl}`);
+  console.log(`SSO url: ${oidcUrl}`);
+  console.debug('Getting Token');
+  // Check a secret was supplied or nothing will work
+  var pattern = new RegExp('^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$', 'i');
+  if (!secret.match(pattern)) {
+    fail('No valid service client secret supplied. Specify a "SECRET" environment variable with the secret for TEST_SERVICE_CLIENT.');
+  }
+
+  // Get a token first to use in the run (only lasts 5 minutes)
+  const oidcPayload = {
+    grant_type: 'client_credentials',
+  };
+
+  // Basic Auth
+  const encodedCredentials = encoding.b64encode(`${client}:${secret}`);
+  const oidcOptions = {
+    headers: {
+      Authorization: `Basic ${encodedCredentials}`,
+      'Content-Type': 'application/x-www-form-urlencoded',
+    },
+  };
+
+
+  let oidcRes = http.post(oidcUrl, oidcPayload, oidcOptions);
+  console.debug(`Response from SSO: ${JSON.stringify(oidcRes, 0, 2)}`);
+  if (!check(oidcRes, {
+    'token response is 200': (r) => r.status === 200,
+    'token response has a token': (r) => r.json("access_token")
+  })
+  ) {
+    fail('could not get an access token from SSO, did you supply a client secret?');
+  }
+  return { token: oidcRes.json("access_token") };
+}
+
+
+// -------------------------------------------------------------------------------------------------
+// 3. VU Code
+// -------------------------------------------------------------------------------------------------
+export default function (data) {
+  console.log(`VU: ${__VU}  -  ITER: ${__ITER}`);
+  console.debug('Running Tests');
+  console.debug(`Data from setup: ${JSON.stringify(data)}`);
+  if (!data.token) {
+    fail('ERROR: No token available from setup step.');
+  }
+
+  const options = {
+    headers: {
+      Authorization: `Bearer ${data.token}`,
+      'Content-Type': 'application/json',
+    },
+  };
+
+  const payload = {
+    bcc: [],
+    body: `CHES url: ${emailUrl}`,
+    bodyType: "html",
+    cc: [],
+    delayTS: 0,
```

Comment:
I think we should have this be commented out for now - when this isn't defined, it'll default to 0 (and instead I think the delayTimestamp field in DB will show up null instead of 0).